example/helm/readme.md

```diff
@@ -9,13 +9,17 @@ If you're convinced this is the place for you then keep reading!
 
 ### Initial Steps
 
+To test the Helm example locally requires:
+
+- k3d > v3.0.0
+- helm > v3.0.0
+
 Create a cluster
 
 ```console
-k3d create --name tempo \
-           --publish 16686:80
+k3d cluster create tempo --api-port 6443 --port "16686:80@loadbalancer"
 
-export KUBECONFIG="$(k3d get-kubeconfig --name='tempo')"
+export KUBECONFIG="$(k3d kubeconfig write tempo)"
```

Comment:
is this line necessary?  it looks like k3d 3.2.x automatically sets up your env in your default kubeconfig and switches to it

Reply:
Probably not -> Removed.